src/Parser/DeathKnight/Blood/CHANGELOG.js

```diff
@@ -7,6 +7,16 @@ import SPELLS from 'common/SPELLS';
 import SpellLink from 'common/SpellLink';
 
 export default [
+  {
+    date: new Date('2018-07-19'),
+    changes: <React.Fragment>Updated <SpellLink id={SPELLS.OSSUARY_TALENT.id} /> to its new reduced value.</React.Fragment>,
```

Comment:
Is this meant to be Foul Bulwark instead of Ossuary? afaik Ossuary didn't change.

Reply:
Yes wierd, idk how i messed that up... fixed